server.py

```diff
@@ -239,6 +241,29 @@ def changePassword():
 	return ok()
 
 
+@reset_limit
+@app.route('/update/password/reset/<username>', methods=['POST'])
+def triggerPasswordChange(username):
+	user = User.get_by_name(username)
+
+	if user and user.email:
+		code_str = util.makeUniqueCode(CODE_SIZE)
+		code = Code.get_by_code(code_str)
+		PasswordReset.create(user=user, code=code)
+
+		link = socket.getfqdn() + 'update/password/reset/' + code
```

Comment:
Wouldn't this make it so the point where we accept the reset code overlaps with the point where we request a reset?

Reply:
You're saying we might have an issue because we have two endpoints that have the same "signature", like this:
```
POST /update/password/reset/<username>
GET  /update/password/reset/<code>
```
The key here is one is a POST and the other is a GET. The difference in HTTP method is what allows these to remain separate.

I suppose there could be an argument made for making one `/update/password/reset/trigger/<username>` and the other `/update/password/reset/confirm/<code>`, but it seems a little redundant, imo.